shielddash/core/static/js/app/api/study-api.js

```diff
@@ -31,9 +31,9 @@ export function getStudies() {
   return axios.get(endpoints.GET_STUDIES, {headers}).then(response => {
     store.dispatch(getStudiesSuccess(response.data));
     return response;
-  }).catch(err => {
-    console.error(err);
-    store.dispatch(getStudiesFailure(err.status));
+  }).catch(response => {
```

Comment:
Yea I called it an error for my own readability earlier but this is a better approach.